### PR TITLE
Rename source.wms.ServerType ➛ source.WMSServerType

### PR DESCRIPTION
**Pull request by marcjansen**
This is part of our ongoing struggle to have one provide per file (e.g. #5794).

Please review.

### CODE REVIEW

**Comment by ahocevar** [comment]
s/struggle/effort/ :smile:

Since we have nothing else in the `ol.source.wms` namespace, it should probably be `ol.source.WMSServerType` in `src/ol/source/wmsservertype.js`.

**Comment by marcjansen** [comment]
Done, bigger changeset, but probably the better change.

File: src/ol/source/wms.jsdoc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-/**
- * @namespace ol.source.wms
```
Comment (ahocevar) [inline src/ol/source/wms.jsdoc]: Have you checked if `ol.source.WMSServerType` is properly listed in the API docs after this change?

Comment (marcjansen) [inline src/ol/source/wms.jsdoc]: I didn't. But since the is nothing in the wms namespace any longer, it should. I cannot try right now... already offline.

Comment (ahocevar) [inline src/ol/source/wms.jsdoc]: You'll have to rebase this pull request anyway before merging. Maybe you get a chance to verify that then.

**Comment by ahocevar** [comment]
Please merge after resolving conflicts and ensuring API docs coverage.

**Comment by marcjansen** [comment]
Will do. Thanks for the review.

**Comment by marcjansen** [comment]
In the TileWMS config options of the constructor:

image

and that link goes to

image

The linked HTML-file is `apidoc/ol.source.html#.WMSServerType`

**Comment by marcjansen** [comment]
I'll merge when green.

**Pull request merged**